make/langtools/netbeans/nb-javac/build.xml

```diff
@@ -117,26 +117,106 @@
 
     <target name="-pre-compile-test" depends="jar"/>
 
-    <target name="zip-nb-javac-sources" depends="clean">
-	<zip basedir="../../../../" destfile="../../../..//nb-javac-${nb-javac-ver}-sources.zip">
-	    <exclude name="**/build/"/>
+    <target name="zip-nb-javac-sources">
+        <zip basedir="../../../../" destfile="dist/nb-javac-${nb-javac-ver}-sources.zip">
+            <exclude name="**/build/"/>
             <exclude name="**/dist/"/>
             <exclude name="**/nbproject/private/"/>
-	    <exclude name="**/nb-javac-sources.zip"/>
-	    <include name="src/jdk.compiler/**"/>
-	    <include name="src/java.compiler/**"/>
-	    <include name="src/jdk.jdeps/**"/>
-	    <include name="make/langtools/netbeans/nb-javac/**"/>
-	    <include name="README.md"/>
-	    <include name="SECURITY.md"/>
-	    <include name="LICENSE.txt"/>
-		<include name=".hgignore"/>
-		<include name=".hgtags"/>
-		<include name="CONTRIBUTING.md"/>
-		<include name="make/langtools/tools/**"/>
-		<include name="make/langtools/build.xml"/>
-		<include name="make/langtools/build.properties"/>
-	
-	</zip>
+            <exclude name="**/nb-javac-sources.zip"/>
+            <include name="src/jdk.compiler/**"/>
+            <include name="src/java.compiler/**"/>
+            <include name="src/jdk.jdeps/**"/>
+            <include name="make/langtools/netbeans/nb-javac/**"/>
+            <include name="README.md"/>
+            <include name="SECURITY.md"/>
+            <include name="LICENSE.txt"/>
+            <include name=".hgignore"/>
+            <include name=".hgtags"/>
+            <include name="CONTRIBUTING.md"/>
+            <include name="make/langtools/tools/**"/>
+            <include name="make/langtools/build.xml"/>
+            <include name="make/langtools/build.properties"/>
+        </zip>
+    </target>
+
+    <!--
+        Publishing ot maven central / OSSRH requires an account from sonatype.
+        The credentials for this account need to be made available to maven
+        using the configured id below.
+
+        With the credentials configured, running
+
+        ant publish-to-ossrh-snapshots
```

Comment:
If the `groupId` is turned into a template, this line would become:
```
ant publish-to-ossrh-snapshots -DgroupId=your.group.id
```
and the line below would be
```
ant publish-to-maven-central -DgroupId=your.group.id
```